typings/lhr.d.ts

```diff
@@ -81,6 +81,23 @@ declare global {
         blockedUrlPatterns: string[];
         extraHeaders: Crdp.Network.Headers;
       }
+
+      export module Audit {
+        export interface OpportunityDetailsItem {
+          url: string;
+          wastedBytes?: number;
+          totalBytes?: number;
+          wastedMs?: number;
+          wastedPercent?: number;
+          fromProtocol?: boolean;
```

Comment:
do we want to just nuke these?

Reply:
> do we want to just nuke these?

yeah, I was building them up to make sure every case was covered, but probably should just be an index type

Reply:
> but probably should just be an index type

turns out it wasn't that simple because we want them to extend lhr-lite, but now an index type + minimum set of properties